### PR TITLE
clipping: revert to previous behaviour to set aspect combobox

### LINKED ISSUE #5326: Crop and rotate doesn't retain the orientation settings when it differs from the image's orientation.
**Describe the bug**
The crop and rotate module doesn't appear to keep the settings from a crop where the orientation of the crop is different to the orientation of the image.

**To Reproduce**

On returning to the Crop and Rotate module the aspect ratio should be the same as applied before existing.

**Screenshots**

cr-set

cr-return

**Platform (please complete the following information):**
 - Darktable Version: Lastest Git pull: 3.1.0+2016~g40a922d0e
Linux - Manjaro
 - OpenCL activated

**Additional context**

### PR DESCRIPTION
**Pull request by elstoc**
Resolves #5326

### CODE REVIEW

**Comment by elstoc** [comment]
I've basically just taken the gui_update code from my working copy of darktable (3.1.0+449~gc5f14d958) in which the functionality worked fine - seems to have resolved the issue.

File: src/iop/clipping.c [hunk 1]
```diff
@@ -1957,7 +1950,19 @@ void gui_update(struct dt_iop_module_t *self)
...
   }
 
-  aspect_presets_changed(g->aspect_presets, self);
+  /* special handling the combobox when current act is already selected
+     callback is not called, let do it our self then..
+   */
+  if(act == -1)
+  {
+    char str[128];
+    snprintf(str, sizeof(str), "%d:%d", abs(p->ratio_d), p->ratio_n);
```
Comment (aurelienpierre) [inline src/iop/clipping.c:1959]: please use `snprintf(str, sizeof(str), "%d:%d %2.2f", abs(p->ratio_d), p->ratio_n, (float)p->ratio_d / (float)p->ratio_n);` to be consistent with `aspect_preset_changed()`

Comment (elstoc) [reply src/iop/clipping.c:1959]: ok

Comment (elstoc) [reply src/iop/clipping.c]: done

**Comment by TurboGit** [comment]
Works for me. Thanks!

**Pull request merged**